mako/cmd.py

```diff
@@ -75,8 +88,19 @@ def cmdline(argv=None):
             raise SystemExit("error: can't find %s" % filename)
         lookup_dirs = options.template_dir or [dirname(filename)]
         lookup = TemplateLookup(lookup_dirs)
+
+        content = None
+        if options.strip_shebang:
```

Comment:
let's replace this pre-opening of the file with a standard preprocessor: https://docs.makotemplates.org/en/latest/usage.html#mako.template.Template.params.preprocessor

Reply:
Thank you, I didn't know that feature - I've updated the code and the stripping is implemented much better now I hope!